insights/parsers/virt_what.py

```diff
@@ -1,17 +1,55 @@
+'''
+VirtWhat - Command ``virt-what``
+================================
+
+Parses the output of the ``virt-what`` command to check if the host is running
+in a virtual machine.
+
+Sample input::
+    kvm
+
+Examples:
+    >>> vw = shared[VirtWhat]
+    >>> vw.is_virtual
+    True
+    >>> vw.generic
+    'kvm'
+    >>> vw.has_specific
+    False
+
+Note:
+    Currently ``virt-what`` fails when running without an environment (or very
+    restricted environment) on RHEL7, and reports below error::
+
+        virt-what: virt-what-cpuid-helper program not found in $PATH
+
+'''
 from .. import Parser, parser
 
+# FIXME:
+# The logic to handle below error should be updated after the bug being fixed.
```

Comment:
@bfahr , @mhuth - Thanks, I refactored this parser and removed the "FIXME" description, please check it again when you are available.